cmd/uwsgi_exporter/main_test.go

```diff
@@ -134,8 +147,18 @@ func testProbe(t *testing.T, data bin) {
 	if err := cmd.Start(); err != nil {
 		t.Fatal(err)
 	}
-	defer cmd.Wait()
-	defer cmd.Process.Kill()
+	defer func() {
+		err := cmd.Wait()
+		if err != nil {
+			slog.Error("uwsgi_exporter exited with error", "error", err)
```

Comment:
```suggestion
            t.Errorf("uwsgi_exporter exited with error: %v", err)
```
Let's fail the test instead of logging an error.